lib/format-message.js

```diff
@@ -1,6 +1,7 @@
 /* global Intl:false */
 'use strict'
 
+var assign = require('lodash/object/assign')
```

Comment:
Do you have an alternative for an `assign()` function? I feel very hesitant to reinvent this wheel just to avoid another dependency.